dask_planner/src/sql/logical/sort.rs

```diff
@@ -0,0 +1,79 @@
+use crate::expression::PyExpr;
+
+use datafusion_expr::logical_plan::Sort;
+pub use datafusion_expr::{logical_plan::LogicalPlan, Expr};
+use pyo3::prelude::*;
+
+#[pyclass(name = "Sort", module = "dask_planner", subclass)]
+#[derive(Clone)]
+pub struct PySort {
+    sort: Sort,
+}
+
+#[pymethods]
+impl PySort {
+    /// Returns a Vec of the sort expressions
+    #[pyo3(name = "getCollation")]
+    pub fn sort_expressions(&self) -> PyResult<Vec<PyExpr>> {
+        let mut sort_exprs: Vec<PyExpr> = Vec::new();
+        for expr in &self.sort.expr {
+            println!("The expr seen is {}", expr);
+            sort_exprs.push(PyExpr::from(expr.clone(), Some(self.sort.input.clone())));
+        }
+        Ok(sort_exprs)
+    }
+    #[pyo3(name = "getColumnName")]
+    pub fn get_column(&self, expr: PyExpr) -> String {
+        match &expr.expr {
+            Expr::Sort {
+                expr,
+                asc,
+                nulls_first,
+            } => {
+                println!("expr {}", expr);
+                println!("asc {}", asc);
+                println!("nulls_first {}", nulls_first);
+                match expr.as_ref() {
+                    Expr::Column(col) => {
+                        println!("Is a column {}", col);
+                        println!("col_name {}", col.name);
+                        col.name.clone()
+                    }
+                    _ => panic!("Sort Expression expr not of column type!"),
+                }
+            }
+            _ => panic!("Provided expression is not a sort expression"),
+        }
+    }
+    #[pyo3(name = "isAscending")]
+    pub fn is_ascending(&self, expr: PyExpr) -> bool {
```

Comment:
It might make sense to return a vector of all bools instead of doing this one by one, but it might be cleaner to have a concrete implementation of a sort based pyexpr that extends some of the standard pyexpr functionality and has additional helpers like these

Reply:
I like having the concrete function. However you could add a sort of "helper" function that accepts and returns a `Vec<bool>` and then internal calls this concrete function.